test/e2e/testdata/template-failed-metric.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: dag-task-test
+spec:
+  entrypoint: dag-task
+  templates:
+  - name: dag-task
```

```suggestion
  entrypoint: dag
  templates:
  - name: dag
```

it's a DAG, not a task, so make the naming clearer